src/db.js

```diff
@@ -0,0 +1,66 @@
+import { createClient } from '@supabase/supabase-js';
+
+// Need access to the database? Message us in the #phlask-data channel on Slack
+const databaseUrl =
+    import.meta.env.VITE_DB_URL || 'https://wantycfbnzzocsbthqzs.supabase.co';
+const resourceDatabaseName =
+    import.meta.env.VITE_DB_NAME || 'resources';
+const databaseApiKey =
+    import.meta.env.VITE_DB_API_KEY || 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6IndhbnR5Y2Zibnp6b2NzYnRocXpzIiwicm9sZSI6ImFub24iLCJpYXQiOjE3MzcwNDY2OTgsImV4cCI6MjA1MjYyMjY5OH0.yczsMOx3Y-zsWu-GjYEajIb0yw9fYWEIUglmmfM1zCY';
```

Comment:
Following up on our standup chat, lets create a sample `.env` file for developers to use in the future so that we can pivot away from storing the key in this file.

We should also update the README to include how to request access to the developer `.env` file in the future.